railties/test/application/configuration_test.rb

```diff
@@ -4324,8 +4328,11 @@ def new(app); self; end
 
     test "ActiveSupport::Cache.format_version is 6.1 by default for upgraded apps" do
       remove_from_config '.*config\.load_defaults.*\n'
+      remove_from_config ".*config.active_support.cache_format_version.*\n"
 
-      app "development"
+      assert_deprecated(ActiveSupport.deprecator) do
```

Comment:
This feels very broad... probably worth matching the expected deprecation message

Reply:
Dropped